eclipse-bluechi/workspace/workloads/smart_trailer_application.kube

```diff
@@ -11,3 +11,6 @@ Yaml=smart_trailer_application.yml
 # Commented to disable the service to automatically start
 [Install]
 WantedBy=default.target
+
+[Unit]
+Wants=agemo.kube
```

Comment:
I think that the dependencies in the sample workloads should be ibeji

All of them interact with Ibeji and Chariott

Reply:
Also should this be .kube or .service?